.github/workflows/run-acceptance-tests.yml

```diff
@@ -164,6 +164,7 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ env.GOVERSION }}
+          check-latest: true
```

Comment:
What does this change give us? We don't normally pass `check-latest: true` so I'm curious why this is necessary here but not in other places.